### PR TITLE
TDR-240 Return the same error for non existent consignment.

### PR DESCRIPTION
**Pull request by MancunianSam**
When we validate that the user owns the consignment, if the consignment
doesn't exist, we return "Invalid consignment Id" and if they don't have
permissions, then we get user xxx does not have accesss...
It's good practice to return the same authorisation error message
whether a resource exists or not to prevent people finding out which
resources exist by checking for the errors. I've changed it so you get
the same message and updated the tests which were checking for it.

### CODE REVIEW

File: src/test/resources/json/addfinaltransferconfirmation_data_error_invalid_consignmentid.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "data": null,
   "errors": [
     {
-      "message": "Invalid consignment id"
+      "message": "User '4ab14990-ed63-4615-8336-56fbb9960300' does not have access to consignment 'f67d1337-cbd0-4fd1-9eac-611489e7113f'"
     }
   ]
 }
```
Comment (TomJKing) [inline src/test/resources/json/addfinaltransferconfirmation_data_error_invalid_consignmentid.json:8]: Missing line break

File: src/test/resources/json/addtransferagreement_data_error_invalid_consignmentid.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "data": null,
   "errors": [
     {
-      "message": "Invalid consignment id"
+      "message": "User '4ab14990-ed63-4615-8336-56fbb9960300' does not have access to consignment 'f67d1337-cbd0-4fd1-9eac-611489e7113f'"
     }
   ]
 }
```
Comment (TomJKing) [inline src/test/resources/json/addtransferagreement_data_error_invalid_consignmentid.json:8]: Missing line break

**Review by TomJKing — COMMENTED** [review]
Looks good. Couple of very minor comments.

File: src/test/resources/json/addfinaltransferconfirmation_data_error_invalid_consignmentid.json (hunk 1 above)
Comment (MancunianSam) [reply src/test/resources/json/addfinaltransferconfirmation_data_error_invalid_consignmentid.json]: I've been trying with the line breaks :smile: I've added this one and the one below.

**Review by TomJKing — APPROVED** [review]

**Pull request merged**